### DISCUSSION TITLE
2 sections in a global header, ok practice?

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1481] [General]
It seems that when I try to have more than one section in a global block, things go downhill fast. So I have to ask, should I expect to be able to have more than one section in a global block? And also, is it expected and OK to have two or more sections of the type "header" in a global block?

Attached is a screenshot showing how the "second header section" in the displayed global block doesn't display any of the blocks that it contains.

![2021-06-24_13-29-47](https://user-images.githubusercontent.com/8103212/123307865-fdc94e00-d4f0-11eb-9841-3190b8237ba2.jpg)

**Comment by eswanzey** [comment]
I tried an alternative whereby one global block contains one section (for the top bar), and another global block contains the another section (for the nav bar). Things seem to act as expected when I do that. Attached is a screenshot.
![2021-06-24_14-16-50](https://user-images.githubusercontent.com/8103212/123313029-0d4b9580-d4f7-11eb-8433-68577d721d21.jpg)

**Comment by mksmanish79** [comment]
Fixed in #1176 #1177
